modules/core/src/main/java/org/apache/fluo/core/util/CuratorUtil.java

```diff
@@ -116,6 +116,14 @@ public static CuratorFramework newCurator(String zookeepers, int timeout, String
     }
   }
 
+  public static boolean pathExist(CuratorFramework curator, String zPath) {
+    try {
+      return curator.checkExists().forPath(zPath) != null;
+    } catch (Exception e) {
```

Comment:
The more specific throw is good, but I think we can have a more specific catch here (and anywhere else where we `catch (Exception e)`. With Java 7 and later, we have the option of enumerating catching multiple checked exceptions if we need to ([multi-catch][1]).

[1]: https://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html

Reply:
If Curator is throwing "Exception", then there's not much we can do, except suggest a patch to Curator.